app/src/main/res/values/strings.xml

```diff
@@ -116,7 +116,7 @@
     <string name="sample_text_steam_url" translatable="false">http://bytefm.cast.addradio.de/bytefm/main/mid/stream</string>
     <string name="sample_text_sleep_timer_remaining_time" translatable="false">00:00</string>
     <!-- Allowed Caller Template -->
-    <string name="allowed_caller_log" translatable="false">
```

Comment:
This string is only visible in the debug log. I would suggest to leave it untranslatable.

Reply:
Failing gracefully is part of doing it right. It is currently annoying because it is in the regular strings, and it looks as if the project isn't fully translated.

Reply:
I am probably making it to easy for myself here, but: This looks like as if Weblate should handle `translatable="false"` differently. :wink:

Reply:
It is this one https://hosted.weblate.org/translate/transistor/strings/en/?checksum=9c1d94904dd3ca43
Not sure why I thought it was handled incorrectly.

Reply:
Not sure either what the problem with that string is. It is actually [part](https://github.com/android/uamp/blob/b36e96a53594b4b346b23c59b70586ea15d6994d/common/src/main/res/values/strings.xml#L32) of some code I copied over from Google's audio playback demo app. I could remove it from strings.xml and put it in code and thus hide it from Weblate.

Reply:
@y20k How about just making it translatable? Of all the strings it is pretty bad to spring that one on users in (not the best) English.

Reply:
> How about just making it translatable?

It just a log message. Basically only visible to developers and/or power users who look at the system logs. For debugging purposes it is best not to have translations of them. For example when the log entry is part of a bug report.

Reply:
@y20k I still like the idea that the whole program is translated. Could change back to translatable if it is ever a problem(?)